### PR TITLE
fix: update pypi package name

### PR DESCRIPTION
**Pull request by skuruppu**
Towards #449

### CODE REVIEW

**Comment by odeke-em** [comment]
I shall later on send docs updates to indicate the new name.

File: setup.py [hunk 1]
```diff
@@ -10,12 +10,11 @@
 
 install_requires = [
     'sqlparse >= 0.3.0',
-    'google-cloud >= 0.34.0',
```
Comment (skuruppu) [inline setup.py:13]: @odeke-em, @busunkim96 asked me to remove this dependency. Hope that's ok with you.

Comment (busunkim96) [reply setup.py:13]: For context, this is an old package (last published July 2018) https://pypi.org/project/google-cloud/. The cloud libraries used to all be published under this single package but were split out.

**Comment by odeke-em** [comment]
What’s the rationale for removing it? It is necessary for folks who’ll be
using this package to install once and not have to manually install
google-cloud as well. I could be wrong though, and if there is another way
to do it then sure.

File: setup.py [hunk 2]
```diff
@@ -10,12 +10,11 @@
...
     'google-cloud-spanner >= 1.8.0',
 ]
 
 setup(
-        name='django-spanner',
+        name='django-google-spanner',
         # Duplicate version here rather than using
```
Comment (busunkim96) [inline setup.py:18]: Please single-source the version. You can treat the version in this file as the source of truth and get the version using `pkg_resources.get_distribution()` in the codebase.

[example in python-spanner](https://github.com/googleapis/python-spanner/blob/55a21d97d0c863cbbbb2d973b6faa4aeba8e38bb/google/cloud/spanner_v1/__init__.py#L19)

File: setup.py (hunk 1 above)
Comment (odeke-em) [reply setup.py:13]: Gotcha, thanks! In that case, can we replace it with the appropriate cloud-Spanner import?

Comment (odeke-em) [reply setup.py:13]: Oh never mind, we already Gabe the google-cloud-spanner dependency. Resolving this conversation as I’ve answered my own question.

File: setup.py (hunk 2 above)
Comment (skuruppu) [reply setup.py]: All done.

**Comment by skuruppu** [comment]
> I think this setup.py would benefit from having a few more fields:
> 
> * `url`
> * `python_requires` (pip won't install the package if the Python version is incompatible)
> * `long_description`
> 
> See [google-cloud-spanner's setup.py](https://github.com/googleapis/python-spanner/blob/master/setup.py#L59-L89) for comparison

I made setup.py look similar to https://github.com/googleapis/python-spanner/blob/master/setup.py.

File: setup.py [hunk 3]
```diff
@@ -6,28 +6,46 @@
...
 from setuptools import find_packages, setup
 
-version = '2.2a0'
 
-install_requires = [
+# Package metadata.
+
+name = "django-google-spanner"
+description = "Bridge to enable using Django with Spanner."
+version = "2.2a0"
```
Comment (skuruppu) [inline setup.py:14]: I assume release please will replace this version since this doesn't look like the version number it's proposing in #453

**Comment by odeke-em** [comment]
If possible I’d highly encourage we stick to the 2.2a0 format that
currently exists because it contains what version of Django (2.2) is
supported but also what version of the release is in (a0)

File: setup.py [hunk 4]
```diff
@@ -39,4 +58,6 @@
             'Framework :: Django',
             'Framework :: Django :: 2.2',
         ],
+        extras_require=extras,
+        python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*",
```
Comment (busunkim96) [inline setup.py:62]: @odeke-em Is this library compatible with 2.7?

**Comment by odeke-em** [comment]
Nope, it is meant for Python 3.5 and above.

**Comment by busunkim96** [comment]
re: versions

I would prefer semver for consistency with other packages. It also makes it easier for folks to pin on this library (e.g., `google-django-spanner>=0.1.0,<1.0.0` If you all decide on some other version schema please explain it clearly in the README. The release bot won't be able to figure out what the correct 'next' release is, so you will have to manually propose releases.

1. Open a PR. In the body of the commit (or the squash merge commit), add 'Release-As: x.x."

```
feat: release 2.2a0

Release-As: 2.2.a0
```

2. Wait for Release-Please to open a new PR.

**Comment by skuruppu** [comment]
@busunkim96, for reducing the maintenance burden on my team, I don't want to deviate from the versioning done by other python packages in googleapis. We want to automate as much of the chores as possible.

@odeke-em, this is how other [googeapis django](https://www.google.com/search?q=django+google+pypi&rlz=1CAOFPC_enAU814&oq=django+google+pypi&aqs=chrome..69i57j0l2j69i60l3j69i64l2.6894j0j7&sourceid=chrome&ie=UTF-8) releases are versioned (if you click through the search links).

File: setup.py (hunk 4 above)
Comment (skuruppu) [reply setup.py]: Updated to >=3.5.

File: setup.py (hunk 3 above)
Comment (timgraham) [reply setup.py:14]: That wouldn't be ideal. The README explains the typical versioning of database backends: "Use the version of django-spanner that corresponds to your version of Django. For example, django-spanner 2.2.x works with Django 2.2.y. (This is the only supported version at this time.)"

Comment (skuruppu) [reply setup.py:14]: @timgraham thanks for that info. But see my [comment](https://github.com/googleapis/python-spanner-django/pull/454#issuecomment-631191745). There are lots of existing django packages that are published by Google and they all consistently use the versioning scheme used by release-please.

Comment (timgraham) [reply setup.py:14]: Yes, sem-var versioning is typical for other Django libraries that often support multiple versions of Django. Database backends are different. The best practice is to create separate release branches for each version of Django that's supported.

Comment (skuruppu) [reply setup.py:14]: I spoke to @larkee and they explained that the manual releasing won't be so painful.

> re: versions
> 
> I would prefer semver for consistency with other packages. It also makes it easier for folks to pin on this library (e.g., `google-django-spanner>=0.1.0,<1.0.0` 

@timgraham do you have any thoughts on the issue of pinning releases when you have non-semver versioning?

Comment (timgraham) [reply setup.py:14]: If using Django 2.2.x, use `django-google-spanner>=2.2,<3.0`. I think `django-google-spanner==2.2.*` would be equivalent.

Comment (skuruppu) [reply setup.py:14]: In that case, @timgraham or @odeke-em could you update the README to explain why the versioning is the way it is? Please include links to the Django README as a reference.

Comment (timgraham) [reply setup.py:14]: What I quoted is from the README in this repo.

File: setup.py [hunk 5]
```diff
@@ -6,28 +6,46 @@
...
     'sqlparse >= 0.3.0',
-    'google-cloud >= 0.34.0',
     'google-cloud-spanner >= 1.8.0',
 ]
+extras = {}
+
+
+# Setup boilerplate below this line.
+
+package_root = os.path.abspath(os.path.dirname(__file__))
```
Comment (larkee) [inline setup.py:29]: You need to `import os`. The presubmit tests are all failing because `os` is not defined

**Pull request merged**